Packages/org.plunderludics.UnityHawk/Runtime/Paths.cs

```diff
@@ -1,77 +1,76 @@
 using System;
 using System.IO;
+using UnityEditor;
 using UnityEngine;
 
 namespace UnityHawk {
 
 public static class Paths
 {
-    public static readonly string packageName = "org.plunderludics.UnityHawk";
-    public static readonly string bizhawkDirRelative = Path.Combine(packageName, "BizHawk~");
-    private static readonly string _bizhawkDirForEditor = Path.Combine("Packages", bizhawkDirRelative);
+    // -- exe --
+    private static readonly string packageName = "org.plunderludics.UnityHawk";
+    public static readonly string BizhawkDirRelative = Path.Combine(packageName, "BizHawk~");
+    private static readonly string _bizhawkDirForEditor = Path.Combine("Packages", BizhawkDirRelative);
     // [^ seems insane but somehow Unity makes this work whether the package is in the Library/PackageCache/ dir or the Packages/ dir
     //  maybe not the best idea to rely on this dark behaviour though..]
 
     // In the build put the bizhawk stuff inside "xx_Data/com.plunderludics.UnityHawk/BizHawk"
-    private static readonly string _bizhawkDirForBuild = Path.Combine(Application.dataPath, bizhawkDirRelative);
-    
-    public static readonly string bizhawkDir =
+    public static string BizHawkDir =>
 #if UNITY_EDITOR
         _bizhawkDirForEditor;
 #else
-        _bizhawkDirForBuild;
+        Path.Combine(Application.dataPath, BizhawkDirRelative);
 #endif
 
-    private static readonly string _emuhawkExeName = 
+    private static readonly string _emuhawkExeName =
 #if UNITY_EDITOR_OSX || UNITY_STANDALONE_OSX
         "EmuHawk-Headless.exe";
 #else
         "EmuHawk.exe";
 #endif
 
-    public static readonly string emuhawkExePath = Path.Combine(bizhawkDir, _emuhawkExeName);
+    public static readonly string emuhawkExePath = Path.Combine(BizHawkDir, _emuhawkExeName);
 
-    public static readonly string defaultConfigPath = Path.Combine(bizhawkDir, "config.ini");
+    public static readonly string defaultConfigPath = Path.Combine(BizHawkDir, "config.ini");
 
-    public static readonly string dllDir = Path.Combine(bizhawkDir, "dll");
+    public static readonly string dllDir = Path.Combine(BizHawkDir, "dll");
+
+    // -- assets --
+    private static readonly string _bizhawkAssetsDirForEditor = Application.dataPath;
+    public static readonly string BizHawkAssetsDirForBuild = Application.streamingAssetsPath;
+
+    static string _bizHawkAssetsDir =>
+#if UNITY_EDITOR
+        _bizhawkAssetsDirForEditor;
+#else
+        Application.streamingAssetsPath;
+#endif
+
+    public static readonly string FirmwarePath = Path.Combine(_bizHawkAssetsDir, "StreamingAssets/Firmware");
+    public static readonly string RamWatchPath = _bizHawkAssetsDir;
+    public static readonly object SavestatesOutputPath = _bizHawkAssetsDir;
```

Comment:
This was also configurable before (as a defaultasset) but i guess i'm fine w regression on that since I never use it
I do think it's better if it's configurable though, even if just as a string

Reply:
Oh wait, i think it's important that the default value is the parent folder of the rom like it was before, saves a lot of time when saving states